olca-proto-io/src/main/java/org/openlca/proto/io/output/ProcessWriter.java

```diff
@@ -61,6 +55,50 @@ public ProtoProcess write(Process process) {
     return proto.build();
   }
 
+	private ProtoProcessDocumentation.Builder mapDocOf(ProcessDocumentation d) {
+		var proto = ProtoProcessDocumentation.newBuilder();
+		if (d == null)
+			return proto;
+
+		proto.setTimeDescription(d.time);
```

Comment:
this will throw an exception, please use `Strings.orEmpty` like in the other writer methods